<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputNum_Widget</class>
 <widget class="QWidget" name="InputNum_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InputNumber</string>
  </property>
  <widget class="QLineEdit" name="Input_LineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>230</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>4</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QSplitter" name="splitter_7">
      <property name="font">
       <font>
        <pointsize>4</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="Button_3">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_6">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_9">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_5">
      <property name="font">
       <font>
        <pointsize>4</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="Button_1">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_4">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_7">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_6">
      <property name="font">
       <font>
        <pointsize>4</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="Button_2">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_5">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_8">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSplitter" name="splitter_8">
      <property name="font">
       <font>
        <pointsize>4</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="Button_0">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Del_Button">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Enter_Button">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>4</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Tips_Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>272</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <pointsize>4</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tips: Press &lt;Enter&gt; to exit input mode</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
